tasks/task02/Area_of_a_triangle.py

```diff
@@ -1,34 +1,39 @@
 # main menu
 import math
-print("Welcome to the triangle area calculation tool.")
+print("\nWelcome to the triangle area calculation tool.")
```

Comment:
by the way, why do you add `\n` in the beginning?

Reply:
for beauty